scripts/build_server_dev

```diff
@@ -26,5 +26,5 @@ bazel_build_flags+=(
 
 # Use a different output_base so that we don't lose incremental state.
 # See https://docs.bazel.build/versions/master/command-line-reference.html#flag--output_base.
-bazel --output_base="$CACHE_DIR/$CONFIG" build "${bazel_build_flags[@]}" //oak/server/dev:oak
+bazel --output_base="$CACHE_DIR/$CONFIG" build "${bazel_build_flags[@]}" //oak/server/dev:oak //oak/server/dev:oak_app_loader
```

Comment:
I think there might be other scripts that need updating too (e.g. `scripts/run_examples_dev`)

Reply:
It will be done in https://github.com/project-oak/oak/pull/474